src/Illuminate/Database/Console/DatabaseInspectionCommand.php

```diff
@@ -224,7 +224,7 @@ protected function ensureDependenciesExist()
     protected function installDependencies()
     {
         $command = collect($this->composer->findComposer())
-            ->push('require doctrine/dbal')
+            ->push('require doctrine/dbal^3.5.1')
```

Comment:
```suggestion
            ->push('require doctrine/dbal^3.5')
```

Reply:
https://github.com/laravel/framework/blob/88b309cd3445cf3d9308d65286ce2997bf1909b8/src/Illuminate/Database/composer.json#L39

Reply:
It's fine 👍